crates/walrus-sdk/src/config/communication_config.rs

```diff
@@ -266,7 +265,7 @@ pub(crate) mod default {
         3
     }
 
-    /// This corresponds to 100Mb, i.e., 1 second on a 100 Mbps connection.
+    // This corresponds to 100Mb, i.e., 1 second on a 100 Mbps connection.
```

Comment:
Why is this `pub`? Could be `pub(crate)`?

Reply:
Ah, that's confusing. I guess I thought that the linter would pick that up as an inconsistency. No worries. Thanks!